src/test/fuzz/coins_view.cpp

```diff
@@ -288,3 +299,22 @@ FUZZ_TARGET(coins_view, .init = initialize_coins_view)
             });
     }
 }
+
+FUZZ_TARGET(coins_view, .init = initialize_coins_view)
+{
+    FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
+    CCoinsView backend_coins_view;
+    TestCoinsView(fuzzed_data_provider, backend_coins_view);
+}
+
+FUZZ_TARGET(coins_view_db, .init = initialize_coins_view)
+{
+    FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
+    auto db_params = DBParams{
+        .path = "",
+        .cache_bytes = 1 << 20, // 1MiB.
+        .memory_only = true,
+    };
+    CCoinsViewDB coins_db{std::move(db_params), CoinsViewOptions{}};
+    TestCoinsView(fuzzed_data_provider, coins_db, /* is_db = */ true);
```

Comment:
nit: in other cases in this file the param hints are written as:
```suggestion
    TestCoinsView(fuzzed_data_provider, coins_db, /*is_db=*/ true);
```